airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -742,10 +742,21 @@ def __init__(
         waiter_max_attempts: int | None = None,
         waiter_delay: int | None = None,
         waiter_countdown: int | None = None,
-        waiter_check_interval_seconds: int = 60,
+        waiter_check_interval_seconds: int,
```

Comment:
This is a breaking change, we should no do that.

```suggestion
        waiter_check_interval_seconds: int | None = None,
```